pipeline/lib/pipeline.ml

```diff
@@ -84,22 +83,22 @@ let pipeline ~slack_path ~conninfo ?branch ?pull_number ~dockerfile ~tmpfs
       @ docker_cpuset_mems
     in
     let run_at = Ptime_clock.now () in
-    let+ output =
-      Docker.pread ~run_args image
-        ~args:
-          [
-            "/usr/bin/setarch"; "x86_64"; "--addr-no-randomize"; "make"; "bench";
```

Comment:
Should we investigate if we can disable ASLR inside docker containers as well?

Reply:
I forgot to add this back. I was running into issues with this because I'm currently using an ARM machine.

Would you mind adding `setarch` and testing this on your machine?